_pages/history.md

```diff
@@ -0,0 +1,100 @@
+[COM interface](https://en.wikipedia.org/wiki/Component_Object_Model).(<strong>[1](https://web.archive.org/web/20040611154239/http://www.clabs.org/wtr/index.cgi?page=/AboutWtr)</strong>) 
+In 2003 Paul Rogers and Bret Pettichord began using this code to 
+test websites and open sourced the project under the name 
+WTR.(<strong>[2](https://github.com/watir/watir-classic/commit/aca359)</strong>) An 
+[Agile XP conference](https://web.archive.org/web/20041013084540/http://www.xpuniverse.com:80/schedule/T15)
+in August of 2004 prompted the team to rename the project to Watir and redesign the code base using 
+[TDD principles](https://en.wikipedia.org/wiki/Test-driven_development), re-using existing code where 
+possible.(<strong>[3](https://web.archive.org/web/20041011114632/http://www.io.com/~wazmo/blog/archives/2004_08.html)</strong>) 
+The first commit with the new name is from September 30,
+ 2004.(<strong>[4](https://github.com/watir/watir-classic/commit/7c7a8a)</strong>) 
+
+Bret was working at Thoughtworks at this time and was introduced
+to the work Jason Huggins was doing with leveraging JavaScript in the browser for testing using
+a program that was initially called JavaScriptTestRunner,
+and would eventually became known as 
+Selenium.(<strong>[5](https://web.archive.org/web/20140519003122/http://rubyforge.org/pipermail/wtr-general/2004-August/000228.html)</strong>) 
+Fun fact: Jason made a list of potential names for his project, but when Thoughtworks needed
+  to present an agenda that included discussing this technology at a conference in Australia, Jason wasn't
+  available to provide the official name. Paul Hammant and Bret looked at the list, 
+  and Bret's background in the QA community caused him to recognize the humor in calling the project
+  Selenium. The primary expensive commercial UI testing solution at the time was owned by 
+  [Mercury Interactive](https://en.wikipedia.org/wiki/Mercury_Interactive) 
+  and Selenium is known to be a [cure for Mercury poisoning](https://www.ncbi.nlm.nih.gov/pubmed/1304229).
+  Jason returned to find his project named for him, but it's certainly proved to be an excellent brand name.
+
+Original versions of Selenium were focused on being a test framework with its own HTML table based 
+Domain Specific Language (DSL). It had some technological limitations and issues with flexibility, 
+as well as offering multiple ways of doing similar things. Its big advantage was its support
+for all of the various browsers that were starting to steal market share from Microsoft. 
+To keep up with Selenium, various groups created browser specific projects for Watir including
+[FireWatir](https://github.com/facebookarchive/watir), 
+[OperaWatir](https://github.com/operasoftware/operawatir), 
+[SafariWatir](https://github.com/redsquirrel/safariwatir), and
+[ChromeWatir](https://github.com/jijeshmohan/chromewatir)).
```

Comment:
There was a "MineralWatir" too, which was someone's first stab at putting the Watir API on top of Selenium (RC, IIRC)